app/assets/stylesheets/_hint.scss

```diff
@@ -0,0 +1,28 @@
+
+.hint-box {
+	position: relative;
+	margin: 1.5% 0;
+	padding: 1.5%;
+	background-color: $wisp;
+
+	.type {
+		position: absolute;
+		top: 1rem;
+		right: 1rem;
+		color: $gray;
+		font-size: $fs-xsmall;
+	}
+
+	.title {
+		margin-bottom: .2em;
+		font-size: $fs-small;
+		font-weight: $fw-bold;
+	}
+
+	.desc {
+		margin-bottom: 0;
+		font-size: $fs-small;
+	}
+}
+
```

Comment:
our WP repos are tabs so i had that set as default. now that i'm working in repos that not have that, i change.